### ISSUE TITLE
Can't seem to match Sublime Text-like syntax highlighting (+ bugs)

### ISSUE THREAD

**Issue by mikeev261** [mshr-h/vscode-verilog-hdl-support#57]
![sample](https://user-images.githubusercontent.com/4935661/50577956-07cad400-0e01-11e9-9d22-5215017eaf56.png)

Hi, 
I'm attempting to use your System Verilog extension for VsCode (+ virtually any Monokai theme) as a replacement for Sublime and its [excellent System Verilog plugin](https://sv-doc.readthedocs.io/en/latest/) and I'm having multiple issues with it: 

1. As you can see with the module instantiation (module_def), the "data" pin gets a blue highlight, but "clk" doesn't. 

2. The module_def and module_instance are completely unhighlighted. 

3. "module" is a completely different color from "endmodule". This doesn't really make sense. 

4. Numbers aren't colored. It would be ideal if they were. 

5. Parameters/Macros/Anything-in-caps should probably be highlighted. 

Any way we can make this extension look closer to the Sublime version?

**Comment by Raamakrishnan** [comment]
Yes, I have these issues as well. The systemverilog syntax highlighting for this extension is very basic and was originally imported from [mshr-h/vscode-systemverilog-support](https://github.com/mshr-h/vscode-systemverilog-support). That sublime text plugin gives a much better syntax highlighting support.

@mshr-h What do you say about us shifting to this better tmLanguage file from Sublime Text SystemVerilog?
https://bitbucket.org/Clams/sublimesystemverilog/src/32a4194bf56a2430eb824c0dadac00f7f2186636/SystemVerilog.tmLanguage?at=default&fileviewer=file-view-default
